waterdata/templates/partials/header.html

```diff
@@ -1,57 +1,13 @@
-<header class="usa-header usa-header-basic" role="banner">
-    <div class="usa-nav-container">
-        <div class="usa-navbar">
-            <div class="usa-logo" id="basic-logo">
-                <em class="usa-logo-text">
-                    <a href="/" title="Home" aria-label="Home">
-                        Water Data For The Nation
-                    </a>
-                </em>
-            </div>
-            <button class="usa-menu-btn">Menu</button>
+<header role="banner">
+    <div class="usgs-banner-container">
+        <div class="usgs-logo">
+            <a href="https://www.usgs.gov"><img alt="USGS Home" src="{{ config.STATIC_ROOT }}/img/USGS_logo.svg" style="width: 100%"/></a>
         </div>
-        <nav role="navigation" class="usa-nav">
-            <button class="usa-nav-close">
-                <img src="{{ config.STATIC_ROOT }}/img/close.svg" alt="close">
-            </button>
-            <ul class="usa-nav-primary usa-accordion">
-                <li>
-                    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="basic-nav-section-one">
-                        <span>Section title</span>
-                    </button>
-                    <ul id="basic-nav-section-one" class="usa-nav-submenu">
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                    </ul>
-                </li>
-                <li>
-                    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="basic-nav-section-two">
-                        <span>Simple terms</span>
-                    </button>
-                    <ul id="basic-nav-section-two" class="usa-nav-submenu">
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                        <li>
-                            <a href="#">Subsection title</a>
-                        </li>
-                    </ul>
-                </li>
-                <li>
-                    <a class="usa-nav-link" href="javascript:void(0)">
-                    <span>Distinct from each other</span>
-                    </a>
-                </li>
+        <nav class="usgs-header-links-container" role="navigation">
+            <ul>
+                <li aria-hidden="true" id="usgs-home-link"><a href="https://www.usgs.gov">USGS Home</a></li>
+                <li><a href="https://answer.usgs.gov/cgi-bin/gsanswers?tmplt=2">Contact USGS</a></li>
```

Comment:
Should this be https://answers.usgs.gov?

Reply:
Thanks. I thought I cut and pasted this correctly but it failed me again!